### PR TITLE
adds event origin to the transform proto

### PR DESCRIPTION
**Pull request by lavabyrd**
https://figmentio.atlassian.net/browse/GRAPH-358

### CODE REVIEW

**Comment by Seb-figment** [comment]
@lavabyrd Can you update your local protoc to v3.20.1 version before generating file to make the github workflow pass

**Comment by lavabyrd** [comment]
> @lavabyrd Can you update your local protoc to v3.20.1 version before generating file to make the github workflow pass

yup i'm trying to at the moment. The problem is the latest build on brew (mac package installer) is 3.19.4, trying to figure out how to build the latest version manually instead - https://formulae.brew.sh/formula/protobuf#default

looks like there's a PR open for it but it hasn't been merged yet - https://github.com/Homebrew/homebrew-core/pull/99843

**Review by Seb-figment — APPROVED** [review]

File: sf/cosmos/transform/v1/transform.proto [hunk 1]
```diff
@@ -11,3 +11,7 @@ message EventTypeFilter {
 message MessageTypeFilter {
   repeated string message_types = 1;
 }
+
+message EventOriginFilter {
+  repeated string event_origin = 1;
```
Comment (pienkowb) [inline sf/cosmos/transform/v1/transform.proto:16]: If we support passing multiple origins here, the name of the field should be `event_origins`.

Comment (lavabyrd) [reply sf/cosmos/transform/v1/transform.proto:16]: good shout. i'll quickly fix it and remerge

Comment (lavabyrd) [reply sf/cosmos/transform/v1/transform.proto:16]: https://github.com/figment-networks/proto-cosmos/pull/6

**Pull request merged**